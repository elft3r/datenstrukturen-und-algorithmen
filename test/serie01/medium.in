100
11 -492 334 1 -1
30 490 629 1 1
7 -685 -484 -1 -1
28 55 679 -1 -1
30 -895 -208 -1 -1
10 905 -242 -1 -1
20 200 711 -1 -1
10 -150 -84 -1 -1
33 480 -722 1 1
9 -298 919 -1 1
28 -68 850 1 1
11 -860 647 -1 1
10 621 -909 -1 1
10 -114 301 -1 -1
5 96 567 -1 -1
25 -187 -694 1 -1
30 -223 127 -1 -1
0 891 653 1 -1
20 -573 -565 -1 -1
29 615 -686 -1 1
21 -235 -807 1 1
32 213 515 1 1
11 829 97 -1 1
24 -26 -523 1 -1
12 -619 762 -1 1
24 757 -378 -1 1
17 50 -570 -1 1
0 762 392 1 -1
12 799 153 -1 1
1 341 -96 1 1
5 758 -381 1 -1
24 -922 139 1 1
31 -80 -14 -1 -1
31 -136 -776 -1 -1
5 -620 738 -1 -1
34 -40 -257 1 1
16 -384 848 -1 1
19 185 885 -1 1
6 171 605 1 -1
28 126 383 -1 1
16 -895 661 1 -1
21 498 334 1 1
33 -326 -715 -1 1
17 -75 334 1 1
1 132 -243 -1 1
0 -32 506 -1 1
34 658 -690 -1 1
27 -780 382 -1 -1
3 -332 -735 -1 -1
0 544 116 -1 1
8 -724 -588 -1 1
5 565 -732 -1 1
1 991 -919 1 1
26 834 -131 -1 1
3 551 -285 -1 1
23 -713 34 1 1
24 -797 -196 -1 -1
6 -67 -11 -1 -1
31 -682 178 -1 -1
2 504 890 -1 -1
7 845 592 1 -1
28 -84 981 1 -1
10 965 1000 1 -1
1 -236 749 1 -1
1 517 -659 1 -1
22 -417 -37 -1 1
23 341 -537 1 -1
7 706 -275 1 -1
12 891 -728 1 -1
0 -238 87 1 -1
13 566 570 -1 -1
20 643 -984 -1 -1
9 -453 30 -1 -1
18 -842 403 -1 1
0 733 -159 1 1
12 785 430 1 1
29 115 -256 -1 1
6 171 -549 1 1
2 348 754 1 1
11 611 -155 -1 -1
15 -693 422 1 -1
17 640 -622 1 -1
14 -281 -1000 -1 1
28 868 -251 -1 -1
10 -251 397 -1 1
1 533 41 -1 1
25 898 -975 -1 1
12 -355 841 -1 -1
22 121 -686 1 1
31 -967 509 -1 -1
32 -336 -709 1 1
1 652 -930 1 -1
6 480 814 1 -1
26 614 -253 1 1
27 645 -141 -1 1
18 981 473 -1 1
10 -373 -209 1 -1
11 -389 388 1 1
11 929 172 1 1
19 232 593 1 1
