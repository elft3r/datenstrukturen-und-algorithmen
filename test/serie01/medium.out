-826
775325370
-484
679
-895
-242
-911
-84
-1499156996
37504
256782926
104883
71109
301
-663
-694
-223
891
1138
-548214859
-10423197
1408584432
-36962
-26
-164819
39220453
-959640
762
49079
-96
1139
-19976602
-14
-776
-118
-1606625079
-1071216
3222145
171
-96972945
-661
7024934
-1809944536
459373
-243
-32
6254164354
-780
-332
544
2936
-5355
-919
78473333
-1121
-11653605
-797
-67
178
-1394
592
-981
-1000
749
-659
-3909175
878
-275
891
-238
570
341
-453
-2386026
733
131785
-168190889
-3537
1102
-456
693
1262
311527
-251
-30369
41
-114787839
-355
-10825280
509
-1996767465
-930
480
15352921
-105993423
344425
209
13137
66403
3078821
